assembl/static2/routes.json

```diff
@@ -32,6 +32,7 @@
   "administration": "${slug}/administration",
   "unauthorizedAdministration": "${slug}/unauthorizedAdministration",
   "adminPhase": "${phase}",
+  "idea": "${slug}/debate/${phase}/theme/${themeId}",
   "post": "${slug}/debate/${phase}/theme/${themeId}/#${element}",
   "themeInPhase": "${slug}/debate/${phase}/theme/${themeId}",
```

Comment:
@amensouissi already created themeInPhase which is exactly the same. You should replace themeInPhase by idea in this case in the navigation code.

Reply:
Cool, I've made this change in 16c999e8c71134401a78d4d46435517b2271d6ac